addons/website/static/src/js/backend/dashboard.js

```diff
@@ -362,9 +364,18 @@ var Dashboard = AbstractAction.extend({
         } else if (this.date_range === 'year') {
             additional_context = {search_default_year: true};
         }
-        this.do_action($action.attr('name'), {
-            additional_context: additional_context,
-            on_reverse_breadcrumb: this.on_reverse_breadcrumb
+        this._rpc({
+            route: '/web/action/load',
+            params: {
+                'action_id': $action.attr('name'),
+            },
+        })
+        .then(function (r) {
+            r.domain = pyUtils.assembleDomains([r.domain, `[('website_id', '=', ${self.website_id})]`])
```

Comment:
wouldn't
```js
 r.domain = pyUtils.assembleDomains([r.domain, [('website_id', '=', self.website_id)]]);
```
work ?

Also, you forgot the `;`

Reply:
No since it has to be list of strings domain
```javascript
@param {string[]} domains list of string representing domains
```

Reply:
alright then :+1: